### ISSUE TITLE
如何自定义Y轴的刻度文字啊

### ISSUE THREAD

**Issue by Maxcucurbit** [AAChartModel/AAChartKit#643]
大神，Y轴的刻度文字可以自定义吗？比如我要实现一个配速图表，那么Y轴文字要显示6’23“、7‘34"、7’02“...
我该如何设置？

**Comment by Maxcucurbit** [comment]
@AAChartModel 大佬，可否帮忙解答一下

**Comment by AAChartModel** [comment]
方便贴一下你的模拟数据和设计稿吗?

**Comment by Maxcucurbit** [comment]
> 方便贴一下你的模拟数据和设计稿吗?

横坐标的数据是[1km,2km,3km,4km,5km]
纵坐标的数据是[7'34'', 8'23'', 8'41'', 7'01'', 9'18'']
现在的遇到的问题：我要把纵坐标显示成这种配速文字，我不知道如何修改，求赐教

**Comment by AAChartModel** [comment]
>  [7'34'', 8'23'', 8'41'', 7'01'', 9'18'']

你这个只是纵坐标的文字,而不是数据吧?!

**Comment by Maxcucurbit** [comment]
> > [7'34'', 8'23'', 8'41'', 7'01'', 9'18'']
> 
> 你这个只是纵坐标的文字,而不是数据吧?!

是的 纵坐标数据是[454, 503, 521, 421, 558] 单位是秒，转出配速文字就是[7'34'', 8'23'', 8'41'', 7'01'', 9'18'']了，不知道该怎么自定义Y轴来达到我要的需求

**Comment by Maxcucurbit** [comment]
@AAChartModel 大佬，求赐教啊

**Comment by AAChartModel** [comment]
https://api.highcharts.com.cn/highcharts#yAxis

这是highCharts自定义Y轴的API文档,目前没有找到实现你的需求的相关接口.

**Comment by AAChartModel** [comment]
Swift项目,或许原生的Charts图表更适合些.

**Comment by Maxcucurbit** [comment]
> Swift项目,或许原生的[Charts](https://github.com/danielgindi/Charts)图表更适合些

项目已经做的差不多了，就差这个图表优化，谢谢了
